guides/source/configuring.md

```diff
@@ -926,6 +926,8 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 - `config.active_storage.replace_on_assign_to_many`: `true`
 - `config.active_record.collection_cache_versioning`: `true`
 
+Note that the method not only does load defaults of a version you specified, but it also loads default configurations of all previous versions.
```

Comment:
I think mapping the "With 'x.x'" with this text is too cumbersome, I've been wondering if we can rephrase the headers to communicate that it's <= 6.0 for instance.

Maybe if we sort them in descending order and say something like "With '6.0', these including previous versions' new defaults"?

Reply:
Yeah, sorting them in descending and rephrasing them to emphasize that "it includes previous versions' new defaults" will definitely be easier to understand. Thanks for the advice.

It would be great to backport to `6-0-stable` to refresh this on https://guides.rubyonrails.org when a new version of 6.0.x is released?